slack/slice_test.go

```diff
@@ -0,0 +1,87 @@
+package slack
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestFindReturnsTrueWhenSliceContainsGivenString(t *testing.T) {
+	cases := []struct {
+		n string
+		h []string
+	}{
+		{"Caninos", []string{"Simularbre", "Caninos", "Noctali", "Gobou"}},
+		{"Caninos", []string{"Caninos"}},
+	}
+
+	for _, testCase := range cases {
```

Comment:
Golang test framework provides an helper in order to run table driven test in a clear manner. 

Have a look [here](https://github.com/golang/go/wiki/TableDrivenTests)

You can write
```
for _, testCase := range cases {
    t.Run(testCase.label, func(t *testing.T) {
        // validation code here
    })
}

```